src/js/tracing/reactnavigationv4.ts

```diff
@@ -203,6 +203,16 @@ class ReactNavigationV4Instrumentation extends InternalRoutingInstrumentation {
     state: NavigationStateV4,
```

Comment:
```suggestion
    state: NavigationStateV4 | undefined,
```
Add to the type that it can also be undefined